examples/next-13/app/CountdownPage.tsx

```diff
@@ -1,94 +1,170 @@
 'use client';
 
+import { PlayerProps } from '@livepeer/react';
+import dayjs from 'dayjs';
+import timezone from 'dayjs/plugin/timezone';
+import utc from 'dayjs/plugin/utc';
+import { ReactNode, useEffect, useState } from 'react';
 import Countdown from 'react-countdown';
 
-interface Props {
-  poster: string;
-  countdown: Date;
-}
+import PlayerPage from './PlayerPage';
+import { fetchPlaybackInfo } from './page';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
 
+interface CountdownPageProps extends PlayerProps<object, any> {
+  countdown: number;
+  id: string;
+}
 interface CountdownProps {
   hours: number;
   minutes: number;
   seconds: number;
   completed: boolean;
 }
 
-export default ({ poster, countdown }: Props) => {
-  const formatDate = (date: Date) => {
-    return date.toLocaleString('en-US', {
-      month: 'long',
-      day: 'numeric',
-      hour: 'numeric',
-      minute: 'numeric',
-      hour12: true,
-    });
+const nycToLocalTimestamp = (timestamp: number): number => {
+  return dayjs.tz(timestamp, 'America/New_York').local().valueOf();
+};
+
+const formattedDate = (timestamp: number): string => {
+  return dayjs(timestamp).format('MMMM D, h:mm a');
+};
+const CountdownPage: React.FC<CountdownPageProps> = ({
+  poster,
+  countdown,
+  id,
+  src,
+  playbackInfo,
+  muted,
+  autoPlay,
+  loop,
+  objectFit,
+  lowLatency,
+}) => {
+  const [startingSoon, setStartingSoon] = useState(!countdown);
+  const [isLive, setIsLive] = useState(false);
+
+  const localCountdown = nycToLocalTimestamp(countdown);
+
+  const onCountdownComplete = async () => {
+    setStartingSoon(true);
+    const interval = setInterval(async () => {
+      try {
+        const playbackData = await fetchPlaybackInfo(id);
+        if (playbackData?.meta?.live) {
+          setIsLive(true);
+          clearInterval(interval);
+        }
+      } catch (error) {
+        console.error('Error fetching playback info:', error);
+      }
+    }, 2000);
```

Comment:
Here we are checking the live status every 2 seconds to check if the stream is live, let me know if there is a better way to do it

Reply:
May want to use `react-query` to avoid over-fetching when the tab / window isn't focused. 

e.g. https://tanstack.com/query/v4/docs/react/examples/react/auto-refetching